.github/workflows/buildDeploy.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and deploy docker image
+
+on: 
+
+  workflow_dispatch: # Allows manual workflow trigger
+  repository_dispatch: # Allows API workflow trigger
+
+  # Triggers the workflow on pull request events but only for the "main" branch
+  # pull_request:
+  #   branches: [ "main" ]
```

Comment:
Are these still needed ?

Reply:
I mean Chewie can also deploy to mainnet, which is id 1, but I guess you're right to, since we have no plans on doing a deployment there anytime soon.

We will deploy on there eventually, but by that time, I expect us to have moved everything over to AWS and deploy the CDapp, which will be a whole different kettle of fish :)